### ISSUE TITLE
"___BEGIN__" in output of make_sentence_with_start()

### ISSUE THREAD

**Issue by quietmint** [jsvine/markovify#90]
With Python 2.7.9 and markovify v0.7.0, when I use `model.make_sentence_with_start()` with `strict=True`, I am getting the literal string `___BEGIN__` as the first word of the sentence. Occasionally I even get this with `strict=False`, but much less often (my corpus text is ~6 MB and many sentences start with and contain the word I requested).

For example, it produced `___BEGIN__ Who is right?` with this:
```
import markovify

text = open('corpus.txt').read()
model = markovify.Text(text, state_size=3)
print(model.make_sentence_with_start('Who'))
```

**Comment by jsvine** [comment]
Strange, and thanks for flagging! Could you share a minimal script and corpus text that reproduces the problem?

**Comment by quietmint** [comment]
I updated the script above. Here's a sample corpus text: https://files.quietmint.ml/corpus.txt

**Comment by jsvine** [comment]
Thanks for flagging this, @quietmint! Looks like it was a bug here:  https://github.com/jsvine/markovify/commit/c41af53ea3b5dfbb693e48a71803f6fd280c71a4#diff-6dd95c17c1c64b93c71d240c9b497656

Fix is in 0.7.1, now available via `pip`. Let me know if it doesn't work for you.

**Comment by quietmint** [comment]
Well that fixed the issue with `___BEGIN__` but now I get `None` as  output sometimes.

This should never happen because there are definitely countless sentences starting with my requested word, and I've set the flag `test_output=False` to ensure any sentence is acceptable (regardless of overlap percentage). On the model, I've also set the flag `retain_original=False` (which should imply `test_output=False` as well, but regardless I've set it explicitly...)
